### DISCUSSION TITLE
Testing on juicefs+elasticsearch, some shards will be in the unassigned state

### DISCUSSION THREAD

**Discussion by yanming-zhang** [juicedata/juicefs#1824] [General]
When testing juicefs+elasticsearch, I encountered a problem, not sure if it is a bug?
It is to write to a certain extent (about 5G or more), some shards will be in the unassigned state

![image](https://user-images.githubusercontent.com/11608532/161095337-68b39309-c6d8-42ee-8d3d-0f0505946f68.png)

**Comment by yanming-zhang** [comment]
The version I use is v1.0.0-beta2

**Comment by davies** [comment]
@yanming-zhang Have you found something related to it in the logging of Elasticsearch or JuiceFS?

**Comment by yanming-zhang** [comment]
> @yanming-zhang Have you found something related to it in the logging of Elasticsearch or JuiceFS?

need to set some parameters, then work normally
attr-cache=0
entry-cache=0
dir-entry-cache=0

**Comment by davies** [comment]
@yanming-zhang We have hit the issue before, and setting attr-cache=0.1 may be enough. It could be caused by a bug in kernel, have not tested it in recent kernel yet.
